File: src/dlls/CSSPModels/srcweb/WebMWQMSite.cs
/*
 * Manually edited
 * 
 */
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPModels
{
    [NotMapped]
    public partial class WebMWQMSite
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<MWQMSiteModel> MWQMSiteModelList { get; set; }
        #endregion Properties

        #region Constructors
        public WebMWQMSite()
        {
            TVItemParentList = new List<WebBase>();
            MWQMSiteModelList = new List<MWQMSiteModel>();
        }
        #endregion Constructors
    }

    public partial class MWQMSiteModel : WebBase
    {
        #region Properties
        public MWQMSite MWQMSite { get; set; }
        public List<MWQMSiteStartEndDate> MWQMSiteStartEndDateList { get; set; }
        #endregion Properties

        #region Constructors
        public MWQMSiteModel()
        {
            MWQMSite = new MWQMSite();
            MWQMSiteStartEndDateList = new List<MWQMSiteStartEndDate>();
        }
        #endregion Constructors
    }
}

File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webmwqmsite.ts
import { MWQMSite } from './generated/MWQMSite.model';
import { MWQMSiteStartEndDate } from './generated/MWQMSiteStartEndDate.model';
import { WebBase } from './webbase';

export class WebMWQMSite {
    TVItemParentList: WebBase[] = [];
    MWQMSiteModelList: MWQMSiteModel[] = [];
}

export class MWQMSiteModel extends WebBase {
    MWQMSite: MWQMSite = new MWQMSite();
    MWQMSiteStartEndDateList: MWQMSiteStartEndDate[] = [];
}

